pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/ModelASTAxisContainer.java

```diff
@@ -30,6 +29,7 @@ public JSONArray toJSON() {
     @Override
     public void validate(@Nonnull ModelValidator validator) {
         // nothing to validate on this element
```

Comment:
Not exactly true any more. =) Same thing on `ModelASTExclude` and `ModelASTExcludes`.